pkg/controller/daemon/controller.go

```diff
@@ -312,6 +312,7 @@ func (dsc *DaemonSetsController) deletePod(obj interface{}) {
 			return
 		}
 	}
+	glog.V(4).Infof("Pod %s deleted.", pod.Name)
```

Comment:
That should hit one of the code paths above, and retrieve the tombstone.obj, which if it isn't a pod should return